### ISSUE TITLE
Create a REST endpoint, per Application Profile, that delivers all Metadata in a RDF/XML format.

### ISSUE THREAD

**Issue by l00mi** [zazuko/cube-creator#823]
To feed opendata.swiss we need to export the metadata in a specific format. The ideal solution is a SPARQL query which response in RDF/XML which is DCAT and it works.

Potentially we need to tweak the RDF/XML to make it actually work.

**Comment by martinmaillard** [comment]
What do you mean by "Application Profile"?

**Comment by l00mi** [comment]
Yes, we need to have an official name for this. I speak about the structure which defines the Endpoint / baseurl and Organisation and such things.

**Comment by l00mi** [comment]
So for a start run a SPARQL query listing all published cubes.

**Comment by tpluscode** [comment]
Why does this have to be RDF/XML specifically?

**Comment by l00mi** [comment]
Becaus that is what ckan. The platform behind opendata.swiss needs. Now lets pray that this is all, I fear the need a XML which is not actually read by a RDF aware component.

**Comment by martinmaillard** [comment]
Separate app for endpoint is here at the moment https://github.com/zazuko/ckan-datasets

**Comment by l00mi** [comment]
closed with https://github.com/zazuko/cube-creator/pull/824
